pkg/cluster/operator.go

```diff
@@ -76,10 +76,16 @@ func OperatorExists(ctx context.Context, cli client.Client, operatorPrefix strin
 // CustomResourceDefinitionExists checks if a CustomResourceDefinition with the given GVK exists.
 func CustomResourceDefinitionExists(ctx context.Context, cli client.Client, crdGK schema.GroupKind) error {
 	crd := &apiextv1.CustomResourceDefinition{}
-	resourceInterval, resourceTimeout := 2*time.Second, 5*time.Second
+	resourceInterval := 161 * time.Millisecond
```

Comment:
we do not need to be that precious to map old value.
could just set factor to 1.0
```
backoff := wait.Backoff{
		Duration: 2 * time.Second,
		Factor:   1.0,
		Steps:    3,
	}
```